### PR TITLE
main skeleton

### PR DESCRIPTION
**Pull request by bketelsen**
This adds a simple http handler in main.go as a starting point.

### CODE REVIEW

**Comment by bketelsen** [comment]
@eduncan911 @alaska @preillyme @erikstmartin reviews please?

**Comment by bketelsen** [comment]
@peterbourgon if you have an opportunity, I'd appreciate your guidance here too.

**Comment by eduncan911** [comment]
Are we going to have a single binary running both the API and public frontend?

File: main.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+}
+func interrupt() error {
+	c := make(chan os.Signal)
+	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
+	return fmt.Errorf("%s", <-c)
+}
+
+type loggingCollector struct{}
+
+func (loggingCollector) Collect(s *zipkin.Span) error {
```
Comment (eduncan911) [inline main.go:126]: Is this the global "log" func we would call from other files?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+			Name:      "duration_nanoseconds_total",
+			Help:      "Total nanoseconds spend serving requests.",
+		}, []string{}),
+	)
+
+	// `package tracing` domain
+	zipkinHost := "my-host"
+	zipkinCollector := loggingCollector{}
+	zipkinAddName := "CONGO" // is that right?
+	zipkinAddSpanFunc := zipkin.NewSpanFunc(zipkinHost, zipkinAddName)
```
Comment (peterbourgon) [inline main.go]: I would just leave the Zipkin stuff out for now. It's only useful if you already have a Zipkin infrastructure in place.

**Comment by bketelsen** [comment]
@eduncan911 It's a possibility, but undecided at this point.

**Pull request merged**